packages/assets-controllers/src/token-prices-service/codefi-v2.test.ts

```diff
@@ -40,20 +117,93 @@ describe('CodefiTokenPricesServiceV2', () => {
         });
 
       expect(pricedTokensByAddress).toStrictEqual({
```

Comment:
It might be good to rename these variables to suit the name of the type here once we've settled on that. So, perhaps `tokensByAddress` or something.